test/integration/datapath/datapath_windows_test.go

```diff
@@ -48,12 +53,35 @@ Timeout context is controled by the -timeout flag.
 func setupWindowsEnvironment(t *testing.T) {
 	ctx := context.Background()
 
+	t.Log("Get REST config")
+	restConfig := kubernetes.MustGetRestConfig(t)
+
 	t.Log("Create Clientset")
 	clientset, err := kubernetes.MustGetClientset()
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	if *initBYOCNI {
+		// create ipmasq configmaps
+		err := kubernetes.MustSetupConfigMap(ctx, clientset, IPMasqAgentCustomConfigMap)
+		require.NoError(t, err, "failed to setup ip masq agent custom configmap")
+		err = kubernetes.MustSetupConfigMap(ctx, clientset, IPMasqAgentReconcileConfigMap)
```

Comment:
If we deploy ipmasqagent here, let's remove the ip-masq-agent installation from pipeline:
https://github.com/Azure/azure-container-networking/blob/master/.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

Line 72:
kubectl apply -f test/integration/manifests/ip-masq-agent/ip-masq-agent.yaml --validate=false

Reply:
Hmm, that's on Linux, and I'm not sure if we should change, that, because in this PR, we're testing Windows
That was already there, if we want to remove it in a different PR, I'm OK with that, but I think for this PR, we should scope this change to Windows tests only

Reply:
Seems like there are a lot of places, where `ip-masq-agent` has been installed as a `kubectl apply -f yaml`
I will not install it in code here, and make a different PR after this, to change all instances of that installation into go-code, in a separate PR